<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Zeitspanne_Dialog</class>
 <widget class="QDialog" name="Zeitspanne_Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>766</width>
    <height>489</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>vaccipy - Zeitspanne</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="0">
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Reset|QDialogButtonBox::Save</set>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <layout class="QGridLayout" name="main_layout">
       <item row="2" column="1">
        <widget class="Line" name="line_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Hier kannst du Uhrzeiten und Tage eingrenzen, an dem ein Termin gebucht werden soll</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <layout class="QGridLayout" name="zeiten_grid">
         <item row="0" column="0">
          <widget class="QFrame" name="uhr_termin_datum_frame">
           <property name="frameShape">
            <enum>QFrame::Box</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Plain</enum>
           </property>
           <layout class="QGridLayout" name="gridLayout_5">
            <item row="7" column="0">
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <property name="leftMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="QCheckBox" name="i_erster_termin_check_box">
                <property name="text">
                 <string>1. Termin</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="i_zweiter_termin_check_box">
                <property name="text">
                 <string>2. Termin</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="5" column="0">
             <widget class="Line" name="line_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Nur Uhrzeiten zwischen</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <layout class="QFormLayout" name="formLayout">
              <item row="0" column="0">
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string>Start:</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_4">
                <property name="text">
                 <string>Ende:</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <layout class="QHBoxLayout" name="horizontalLayout">
                <item>
                 <widget class="QTimeEdit" name="i_end_time_qtime">
                  <property name="time">
                   <time>
                    <hour>23</hour>
                    <minute>59</minute>
                    <second>0</second>
                   </time>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="uhr_end_label">
                  <property name="text">
                   <string>Uhr</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item row="0" column="1">
               <layout class="QHBoxLayout" name="horizontalLayout_2">
                <item>
                 <widget class="QTimeEdit" name="i_start_time_qtime">
                  <property name="currentSection">
                   <enum>QDateTimeEdit::HourSection</enum>
                  </property>
                  <property name="calendarPopup">
                   <bool>false</bool>
                  </property>
                  <property name="time">
                   <time>
                    <hour>0</hour>
                    <minute>1</minute>
                    <second>0</second>
                   </time>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="uhr_start_label">
                  <property name="text">
                   <string>Uhr</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item row="6" column="0">
             <widget class="QLabel" name="termine_anwenden_label">
              <property name="text">
               <string>Bei welchen Terminen sollen die Bedingungen gelten?</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <layout class="QFormLayout" name="formLayout_2">
              <item row="0" column="0">
               <widget class="QLabel" name="kalender_label">
                <property name="text">
                 <string>Termin fr√ºhestens ab:</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QDateEdit" name="i_start_datum_qdate">
                <property name="alignment">
                 <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                </property>
                <property name="buttonSymbols">
                 <enum>QAbstractSpinBox::UpDownArrows</enum>
                </property>
                <property name="specialValueText">
                 <string/>
                </property>
                <property name="time">
                 <time>
                  <hour>0</hour>
                  <minute>0</minute>
                  <second>0</second>
                 </time>
                </property>
                <property name="currentSection">
                 <enum>QDateTimeEdit::DaySection</enum>
                </property>
                <property name="calendarPopup">
                 <bool>true</bool>
                </property>
                <property name="date">
                 <date>
                  <year>2021</year>
                  <month>5</month>
                  <day>25</day>
                 </date>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="1" column="0">
             <widget class="Line" name="line_5">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="QLabel" name="header_label">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>vaccipy</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <layout class="QGridLayout" name="wochentage_grid">
         <item row="0" column="0">
          <widget class="QFrame" name="tage_frame">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="frameShape">
            <enum>QFrame::Box</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Plain</enum>
           </property>
           <layout class="QGridLayout" name="gridLayout_4">
            <item row="3" column="0" colspan="2">
             <widget class="QCheckBox" name="i_di_check_box">
              <property name="text">
               <string>Dienstags</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
              <property name="weekday" stdset="0">
               <string>Di</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0" colspan="2">
             <widget class="QCheckBox" name="i_mo_check_box">
              <property name="text">
               <string>Montags</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
              <property name="weekday" stdset="0">
               <string>Mo</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0" colspan="2">
             <widget class="Line" name="line_4">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item row="5" column="0" rowspan="2" colspan="2">
             <widget class="QCheckBox" name="i_do_check_box">
              <property name="text">
               <string>Donnerstags</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
              <property name="weekday" stdset="0">
               <string>Do</string>
              </property>
             </widget>
            </item>
            <item row="9" column="0" colspan="2">
             <widget class="QCheckBox" name="i_sa_check_box">
              <property name="text">
               <string>Samstags</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
              <property name="weekday" stdset="0">
               <string>Sa</string>
              </property>
             </widget>
            </item>
            <item row="11" column="0" colspan="2">
             <widget class="QCheckBox" name="i_so_check_box">
              <property name="text">
               <string>Sonntags</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
              <property name="weekday" stdset="0">
               <string>So</string>
              </property>
             </widget>
            </item>
            <item row="4" column="0" colspan="2">
             <widget class="QCheckBox" name="i_mi_check_box">
              <property name="text">
               <string>Mittwochs</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
              <property name="weekday" stdset="0">
               <string>Mi</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0" colspan="2">
             <widget class="QLabel" name="wochentage_label">
              <property name="text">
               <string>Nur an folgenden Tagen:</string>
              </property>
             </widget>
            </item>
            <item row="7" column="0" colspan="2">
             <widget class="QCheckBox" name="i_fr_check_box">
              <property name="text">
               <string>Freitags</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
              <property name="weekday" stdset="0">
               <string>Fr</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>i_mo_check_box</tabstop>
  <tabstop>i_di_check_box</tabstop>
  <tabstop>i_mi_check_box</tabstop>
  <tabstop>i_do_check_box</tabstop>
  <tabstop>i_fr_check_box</tabstop>
  <tabstop>i_sa_check_box</tabstop>
  <tabstop>i_so_check_box</tabstop>
  <tabstop>i_start_datum_qdate</tabstop>
  <tabstop>i_start_time_qtime</tabstop>
  <tabstop>i_end_time_qtime</tabstop>
  <tabstop>i_erster_termin_check_box</tabstop>
  <tabstop>i_zweiter_termin_check_box</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
